charts/collabora-online/2.0.0/questions.yaml

```diff
@@ -19,15 +19,39 @@ groups:
 portals:
   web_portal:
     protocols:
-      - "https"
+      - "$kubernetes-resource_configmap_portal_protocol"
     host:
-      - "$node_ip"
+      - "$kubernetes-resource_configmap_portal_host"
     ports:
-      - "$variable-service.port.nodePort"
-    path: "/loleaflet/dist/admin/admin.html"
+      - "$kubernetes-resource_configmap_portal_port"
 
```

Comment:
Isn't possible to set the path on portals?
This app have the API on `/` and webui portal at `/loleaflet/dist/admin/admin.html`

Reply:
Thanks, for the note!

Reply:
I've adapted the chart, please check again :)

Reply:
Works now! :)

Reply:
Yes it does! 
Just a note, Collabora, with the default options i/we have right now, only works with https (and ssl).
If someone doesn't enable ingress, Collabora won't work
User will have to remove `-o:ssl.termination=true -o:ssl.enable=false` and portal should be changed to http in this case. 

Could be possible in the future to have a question so users can choose between http and https in cases that depends on the configuration? Anyway the dynamic part works for collabora!